doc/languages-frameworks/emscripten.section.md

```diff
@@ -47,6 +47,7 @@ A special requirement of the `pkgs.buildEmscriptenPackage` is the `doCheck = tru
 
 * Use `export EMCC_DEBUG=2` from within a emscriptenPackage's `phase` to get more detailed debug output what is going wrong.
 * ~/.emscripten cache is requiring us to set `HOME=$TMPDIR` in individual phases. This makes compilation slower but also makes it more deterministic.
+* Additionally, during the `configurePhase` of `emscriptenStdenv`, a local (writable) directory is created at `$(pwd)/.emscriptencache`, and the environment variable `EM_CACHE` is set to that path. If overriding the `configurePhase` for a derivation, this step will need to be included manually as well.
```

Comment:
What does `$(pwd)` refer to here?

Reply:
It's certainly possible I should be phrasing it better -- I took it perhaps too literally from https://github.com/NixOS/nixpkgs/pull/172207/files#diff-de4924a70c218dfafd242eda61aaffb33790c7b14b90eed4bf9e3c3aaf2db37aR28-R30, but in reality it's just in whatever directory `emconfigure` would be running inside. Perhaps it's okay to just remove the `$(pwd)/`, since that's more bash-related.

Reply:
Ah, seeing the code helps a lot. That note mostly describes what the code does at a level of detail which equals the code, which I find problematic. Documentation should describe what the user should do to succeed, and only describe the code as far as necessary to understand why. Also, multiple packages set `EM_CACHE`, so that part is not correct.

I suggest removing the new line and changing the above one (see https://github.com/NixOS/nixpkgs/pull/181363/files/08d63f8b6907b679d473f068efd49e567f8b0858#r921869466).

```suggestion
```

I cannot make suggestions for the other lines. Could you please
- delete l. 39-44
- add an introductory sentence to the bullet list, something to the effect of "building emscripten packages requires some special considerations to be aware of, see `pkgs/top-level/emscripten-packages.nix` for details" (with a permalink to the current master commit)
- make the `doCheck = true` line part of the list instead of a sentence and reword to "`pkgs.buildEmscriptenPackage` sets `doCheck = true` by default, which requires each `emscriptenPackage` to have a `checkPhase`.

Reply:
Updated with latest force push @fricklerhandwerk. Let me know if it makes more sense now!